project_task_code_portal/tests/test_portal.py

```
@@ -0,0 +1,279 @@
+            [
+                ("partner_ids", "in", self.partner_portal.id),
+                ("model", "=", "project.project"),
+                ("res_id", "=", self.task_1.project_id.id),
+            ],
+            order="id DESC",
+            limit=1,
+        )
+
+        share_link = str(message.body.split('href="')[1].split('">')[0])
+        match = re.search(
+            r"access_token=([^&]+)&amp;pid=([^&]+)&amp;hash=([^&]*)", share_link
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)